test/cmdlineTests/external_function_pointers_taking_calldata_args/input.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
This test is not really related to function pointers. We're only adding it in this PR because it happened to reveal that we're missing coverage.

My suggestion: `external_function_pointers_taking_calldata_args/` -> `standard_urls_existing_and_missing`